.storybook/stories/useCameraShake.stories.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react'
+import * as THREE from 'three'
+import { Camera, useFrame } from 'react-three-fiber'
+import { withKnobs, button } from '@storybook/addon-knobs'
+
+import { Setup } from '../Setup'
+
+import { useCameraShake, ShakeConfigPartial, ShakeController, PerspectiveCamera } from '../../src'
+
+export default {
+  title: 'Misc/useCameraShake',
+  component: useCameraShake,
+  decorators: [
+    (storyFn) => (
+      <Setup cameraPosition={new THREE.Vector3(0, 0, 5)} controls={false}>
+        {storyFn()}
+      </Setup>
+    ),
+    withKnobs,
+  ],
+}
+
+function StoryBookKnobs({ shaker }: { shaker: ShakeController }) {
```

Comment:
I'm not familiar with Storybook controls, but does it have to be rendered as a component? Couldn't it just be a func?

Reply:
I think your right, could just be a func.